Fix license info

TheHaf-at-FraunhoferIOSB opened the pull request:
This PR fixes the issues discussed in #326. It replaces the contents of the LICENSE file with the contents of the doc/LICENSE file and updates the license information in the POM.

TheHaf-at-FraunhoferIOSB commented:
Is there anything further that I need to do so that this PR can be processed?

rnc commented:
@jwillemsen Can you review this please as well?

jwillemsen commented:
Keeping multiple license files in the repository seems risky to me, is asking for problems in the future. What about removing doc/LICENSE completely, it is now a duplicate of LICENSE, correct? For the pom.xml, maybe use https://raw.githubusercontent.com/JacORB/JacORB/master/LICENSE as link, that is the one that is part of the code, the text on the website could be outdated again at some point.

TheHaf-at-FraunhoferIOSB commented:
I have now removed the doc/LICENSE file and updated the link as suggested.

Sorry for the force-pushes, I had unforseen trouble catching up to the HEAD of master. Should all be ok now.

jwillemsen commented:
The readme in the root directory needs a small update for the link to the license file

TheHaf-at-FraunhoferIOSB commented:
> The readme in the root directory needs a small update for the link to the license file

Thanks, I updated the link.
